packages/google_maps_flutter/google_maps_flutter/example/integration_test/google_maps_test.dart

```diff
@@ -57,14 +66,13 @@ void main() {
       ),
     ));
 
-    compassEnabled = await inspector.isCompassEnabled();
+    compassEnabled = await inspector.isCompassEnabled(mapId: mapId);
     expect(compassEnabled, true);
```

Comment:
Some (many) of these expectations are going to have a special case for the web. For example, the web won't have a "compass" until this lands: https://github.com/flutter/plugins/pull/4916

Reply:
We can always conditionalize tests by platform as necessary when we endorse web.